website/static/css/custom.css

```diff
@@ -0,0 +1,16 @@
+/* your custom css */
+
+@media only screen and (min-device-width: 360px) and (max-device-width: 736px) {
```

Comment:
is it necessary to have this empty media queries?

Reply:
I think he explained it in a previous PR that we'll leave this css files empty because we still don't have specific styles for the docs

Reply:
Yes; that is the reason @fxisco